modules/mono/glue/GodotSharp/GodotSharp/Core/Array.cs

```diff
@@ -25,16 +25,30 @@ protected override bool ReleaseHandle()
         }
     }
 
+    /// <summary>
+    /// Wrapper around Godot's Array class, an array of Variant
+    /// typed elements allocated in the engine in C++. Useful when
+    /// interfacing with the engine. Otherwise prefer .NET collections
+    /// such as <see cref="System.Array"/> or <see cref="List{T}"/>.
```

Comment:
Did `List{T}` resolve correctly here without needing to specify the namespace?

Reply:
Yes, it resolves to `System.Collections.Generic.List<T>`.